src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +22,27 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    private static final int MAX_COUNT_CHAR = 5;
+    private int character;
+    private int currentCharacter = 1;
+
+    public String getUniqueCharacters(String fileName) throws IOException {
+        Set<Character> letters = new TreeSet<>();
+        StringBuilder result = new StringBuilder();
+        try (FileReader reader = new FileReader(fileName)) {
+            while ((character = reader.read()) != -1) {
+                if (Character.isLetter(character)) {
+                    letters.add(Character.toLowerCase((char) character));
+                }
+            }
+            for (Character value : letters) {
+                if (currentCharacter > MAX_COUNT_CHAR) {
```

Comment:
Maybe it would be better to have the condition here as 
`if (result.length() == 5) {`
because in your code you're creating variables `character` and `MAX_COUNT_CHAR` that will not be used if there's a problem with the file